src/main/java/com/aws/greengrass/FleetProvisioningByClaimPlugin.java

```diff
@@ -155,10 +155,6 @@ public ProvisionConfiguration updateIdentityConfiguration(ProvisionContext provi
                 certificateOwnershipToken = response.certificateOwnershipToken;
             } else {
                 Path csrFile = Paths.get(csrPath);
-                if (!csrFile.toFile().exists()) {
```

Comment:
I removed the file exist check section completely, since it is handled by IOException catch below.